packages/netlify-cms-app/src/extensions.js

```diff
@@ -1,4 +1,14 @@
+// Core
```

Comment:
Replacing the type specific extension files with a single module for simplicity.